WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -188,4 +189,22 @@ public static void convertToLowercase(Editable s) {
             s.replace(0, s.length(), lowerCase);
         }
     }
+
+    @NonNull
+    public static Set<String> planTags() {
+        String[] extraFields = {"plan_product_id"};
+        List<Map<String, Object>> blogs = WordPress.wpDB.getBlogsBy("dotcomFlag=1", extraFields);
+        Set<String> tags = new HashSet<>();
+
+        for (Map<String, Object> blog : blogs) {
+            Integer planId = MapUtils.getMapInt(blog, "plan_product_id");
```

Comment:
It seems that there is no need to box the return value of `MapUtils.getMapInt()` into an `Integer`. Unless I'm missing something, let's change the type of `planId` to plain `int`, WDYT?